config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "/"
+baseURL = "https://docs.chef.io/"
```

Comment:
Does this break the local build?

Reply:
I expected it to, but it did not. However, it also didn't create a canonical URL.

Reply:
Not that this is in the Hugo docs anywhere, but the Hugo server knows if it's a local build and will use localhost instead of baseURL.